western/index.json

```diff
@@ -2085,5 +2085,501 @@
     "412:411 M+ 20:34:00 +19:30:00 20:34:00 +20:30:00 VUL DEL",
     "410:690 M+ 20:15:00 +20:30:00 20:15:00 +21:15:00 VUL SGE",
     "687:686 M+ 19:15:00 +19:10:00 19:15:00 +21:05:00 VUL SGE"
+  ],
+  "asterisms": [
+    {
+      "id": "RH1",
+      "is_ray_helper": true,
+      "lines": [[67301, 69673, 65474]]
+    },
+    {
+      "id": "RH2",
+      "is_ray_helper": true,
+      "lines": [[53910, 54061, 11767, 113881, 113963, 113368]]
+    },
+    {
+      "id": "RH3",
+      "is_ray_helper": true,
+      "lines": [[65378, 11767, 6686]]
+    },
+    {
+      "id": "RH4",
+      "is_ray_helper": true,
+      "lines": [[25336, 21421, 17847]]
+    },
+    {
+      "id": "RH5",
+      "is_ray_helper": true,
+      "lines": [[25930, 26311, 26727, 32349]]
+    },
+    {
+      "id": "RH6",
+      "is_ray_helper": true,
+      "lines": [[102098, 59774, 58001, 49669]]
+    },
+    {
+      "id": "RH7",
+      "is_ray_helper": true,
+      "lines": [[58001, 54061, 15863]]
+    },
+    {
+      "id": "RH8",
+      "is_ray_helper": true,
+      "lines": [[59774, 54061, 24608]]
+    },
+    {
+      "id": "RH9",
+      "is_ray_helper": true,
+      "lines": [[59774, 53910, 36850]]
+    },
+    {
+      "id": "RHA",
+      "is_ray_helper": true,
+      "lines": [[59774, 11767, 746, 677]]
+    },
+    {
+      "id": "GDi",
+      "common_name": {"english": "Great Diamond"},
+      "is_ray_helper": false,
+      "lines": [[69673, 65474, 57632, 63125, 69673]]
+    },
+    {
+      "id": "SpT",
```

Comment:
Maybe we should adopt a more unique ID, like "AST Western SpT" ?

Reply:
Doesn't the context already make this unique enough? Does Stellarium Mobile keep these strings in a common database and try to use them as global handles?

Reply:
Yes, for constellations it's used as one of the searchable identifiers